src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -660,6 +660,10 @@ private enum SequenceType
 
     internal static class ReadOnlySequence
     {
+        /// <summary>
+        /// Flag that allow to encode the <see cref="ReadOnlySequence{T}.SequenceType"/>.
```

Comment:
Small typo / grammar: should probably be "Flag that allows encoding..."

Reply:
Fixed.